src/static_canvas.class.ts

```diff
@@ -752,7 +752,7 @@ import { pick } from './util/misc/pick';
       this.cancelRequestedRender();
       this.calcViewportBoundaries();
       this.clearContext(ctx);
-      fabric.util.setImageSmoothing(ctx, this.imageSmoothingEnabled);
+      ctx.imageSmoothingEnabled = this.imageSmoothingEnabled;
       ctx.patternQuality = 'best';
```

Comment:
should this [`patternQuality`](https://github.com/Automattic/node-canvas#canvasrenderingcontext2dpatternquality) be set on Image as well? And all cache canvases?
And maybe [`quality`](https://github.com/Automattic/node-canvas#canvasrenderingcontext2dquality) as well?

Reply:
Those are node-canvas specific things i m ignoring on purpose. Are their default not good?

I want to have a clean canvas interface as much as possible in case we move to skia-canvas or maybe we have both as an option.

Reply:
good is default
I changed that in a PR to fix a visual test I recall

Reply:
but missed `quality`